tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -341,6 +341,7 @@ private static void GenerateCoverageFile(ApiMetadata api, string directory)
             var targetExecutable = new XElement("TargetExecutable", "/Program Files/dotnet/dotnet.exe");
             var targetArguments = new XElement("TargetArguments",
                 $"test --no-build -f {GetPreferredCoverageFramework(api)} -c Release");
+            var filters = new XElement("Filters", new XElement("IncludeFilters", new XElement("FilterEntry", new XElement("ModuleMask", api.Id))));
```

Comment:
Hmm... that sounds like definitely the right call for Spanner, but not necessarily right for other things. Never mind - will do it for now and we can see.